spec/features/number_fields_spec.rb

```diff
@@ -61,12 +88,13 @@
     end
 
     VALID_MONEY_VALUES.each do |pair|
-      it "allows me to enter '#{pair[0]}'" do
-        fill_in "employment-form-national-insurance-field", with: pair[0]
+      it "allows me to enter '#{pair[:input]}'" do
+        fill_in "employment-form-national-insurance-field", with: pair[:input]
         click_on "Save and continue"
+        expect(session_contents["national_insurance"]).to eq pair[:input].to_s.delete(",").delete("£").to_f
```

Comment:
You're right, @willc-work , this is not hugely helpful. Try adding a new `stored` key to each of the test hashes, e.g:

```ruby
{
  input: "  £2000",
  output: "2,000",
  stored: 2000.0,
}

...

expect(session_contents["national_insurance"]).to eq pair[:stored]

```